assets/js/editor.js

```diff
@@ -106,6 +106,7 @@ $gp.editor = (
 					.on( 'click', 'button.insertnl', $gp.editor.hooks.newline )
 					.on( 'click', 'button.approve', $gp.editor.hooks.set_status_current )
 					.on( 'click', 'button.reject', $gp.editor.hooks.set_status_rejected )
+					.on( 'click', 'button.changesrequested', $gp.editor.hooks.set_status_changesrequested )
```

Comment:
If the `gp-translation-helpers` plugin is not active or not on the https://github.com/GlotPress/gp-translation-helpers/pull/65 brnach, then this causes this error message and breaks most of the JS on the page:
```
Uncaught TypeError: can't assign to property "guid" on "button.changesrequested": not an object
```

Reply:
A fix would be to add a stub like this below:
```
				set_status_changesrequested: function() {
					return false;
				},
```

Reply:
Good catch. Resolved [here](https://github.com/GlotPress/GlotPress/pull/1451/commits/65147031bd25e43f2c15a24e51887565fd252a96).